eslint.config.mjs

```diff
@@ -376,6 +379,7 @@ export default [
     files: [
       'packages/salesforcedx**/test/jest/**/*',
       'packages/salesforcedx**/test/unit/**/*',
+      'packages/salesforcedx**/src/__tests__/**/*',
```

Comment:
this should cover both `src/__tests__` AND `packages/salesforcedx-lwc-language-server/src/javascript/__tests__/fixtures/**`, right?
```suggestion
      'packages/salesforcedx**/src/**/__tests__/**/*',
```